src/StimulusBundle/config/services.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\UX\StimulusBundle\AssetMapper\ControllersMapGenerator;
+use Symfony\UX\StimulusBundle\AssetMapper\StimulusLoaderJavaScriptCompiler;
+use Symfony\UX\StimulusBundle\Helper\StimulusHelper;
+use Symfony\UX\StimulusBundle\Twig\StimulusTwigExtension;
+use Symfony\UX\StimulusBundle\Ux\UxPackageReader;
+use Twig\Environment;
+
+use function Symfony\Component\DependencyInjection\Loader\Configurator\abstract_arg;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\param;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
+
+return static function (ContainerConfigurator $container): void {
+    $container->services()
+        ->set('stimulus.helper', StimulusHelper::class)
+            ->arg('$twig', service(Environment::class)->nullOnInvalid())
```

Comment:
better not force using reflection when we can avoid it:
```suggestion
            ->args([service(Environment::class)->nullOnInvalid()])
```